Makefile

```diff
@@ -282,12 +282,9 @@ doc/source/api/mlos_bench/modules.rst: $(MLOS_BENCH_PYTHON_FILES) $(COMMON_DOC_F
 	rm -rf doc/source/api/mlos_bench
 	cd doc/ && conda run -n ${CONDA_ENV_NAME} sphinx-apidoc -f -e -M -o source/api/mlos_bench/ ../mlos_bench/ ../mlos_*/setup.py
 	# Save the help output of the mlos_bench scripts to include in the documentation.
-	conda run -n ${CONDA_ENV_NAME} mlos_bench/mlos_bench/run_opt.py --help > doc/source/api/mlos_bench/mlos_bench.run_opt.usage.txt
-	echo ".. literalinclude:: mlos_bench.run_opt.usage.txt" >> doc/source/api/mlos_bench/mlos_bench.run_opt.rst
-	echo "   :language: none" >> doc/source/api/mlos_bench/mlos_bench.run_opt.rst
-	conda run -n ${CONDA_ENV_NAME} mlos_bench/mlos_bench/run_bench.py --help > doc/source/api/mlos_bench/mlos_bench.run_bench.usage.txt
-	echo ".. literalinclude:: mlos_bench.run_bench.usage.txt" >> doc/source/api/mlos_bench/mlos_bench.run_bench.rst
-	echo "   :language: none" >> doc/source/api/mlos_bench/mlos_bench.run_bench.rst
+	conda run -n ${CONDA_ENV_NAME} mlos_bench/mlos_bench/run.py --help > doc/source/api/mlos_bench/mlos_bench.run.usage.txt
```

Comment:
https://github.com/microsoft/MLOS/pull/313/files#diff-fe3fc947c54d13b7a130147b2331b17caad55820ffead76b85fd38dff3397208R69-R73
and
https://github.com/microsoft/MLOS/pull/313/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R291

has a changes that will distribute this script as "mlos_bench-script_name".
Do you want to slurp that change in here, or leave that for another PR?

Basically we could make it so that `mlos_bench-run` was a command on the user's `PATH` could potentially be convenient and ergonomic.

Reply:
if you don't mind, I would do it in the next PR. This way I can continue working on my stuff without much interference